lib/addCesiumRTC.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+var DeveloperError = Cesium.DeveloperError;
+var Ellipsoid = Cesium.Ellipsoid;
+var defaultValue = Cesium.defaultValue;
+var defined = Cesium.defined;
+
+var addExtensionsUsed = require('./addExtensionsUsed');
+
+module.exports = addCesiumRTC;
+
+/**
+ * Positions the model relative to a center point using the CESIUM_RTC extension.
+ * Either specify options.position directly or calculate it from longitude, latitude and height
+ * along with an ellipsoid (WGS84 if not specified).
+ *
+ * @param {Object} gltf A javascript object holding a glTF hierarchy.
+ * @param {Object} [options] Defines the relationship with the center position
+ * @param {Cartesian3} [options.position] Places the model relative to a particular position.
+ * @param {Number} [options.longitude] The longitude to map onto the ellipsoid.
```

Comment:
We are usually explicit that longitude/latitude is in radians and height is in meters.